lara-typescript/src/interactive-api-client/api.ts

```diff
@@ -388,3 +391,67 @@ export const getInteractiveSnapshot = (options: IGetInteractiveSnapshotOptions)
 export const getLibraryInteractiveList = (options: IGetLibraryInteractiveListRequest) => {
   THROW_NOT_IMPLEMENTED_YET("getLibraryInteractiveList");
 };
+
+export const writeAttachment = (params: Omit<IWriteAttachmentRequest, "requestId" | "operation">) => {
+  return new Promise<void>((resolve, reject) => {
+    const client = getClient();
+    const { name, content, options } = params;
+    const request: IAttachmentUrlRequest = { name, operation: "write", requestId: client.getNextRequestId() };
+    client.addListener("attachmentUrl", async (response: IAttachmentUrlResponse) => {
+      if (response.url) {
+        try {
+          await fetch(response.url, { ...options, method: "PUT", body: content });
+          resolve();
```

Comment:
Perhaps this can return the result of the put? That way if there is some info in result clients would have access to it. Similar to how it is returned in readAttachment.